utils/measure/measure.py

```diff
@@ -299,7 +395,38 @@ def should_resume(self, csv_file_path: str) -> bool:
 
 
     def get_resume_variation(self, csv_file_path: str) -> Variation:
-        """Determine the variation where we have to resume the measurements"""
+        """This method returns the variation to resume at.
+
+        It reads the last row from the CSV file and converts it into a Variation object.
+
+        Parameters
```

Comment:
Parameters and returns are self explanatory, don't need additional comments imo.

Reply:
See https://github.com/bramstroker/homeassistant-powercalc/pull/951/files#r939639547